nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-edit-layout.html

```diff
@@ -60,7 +60,6 @@
       name="subjects"
       value="{{document.properties.dc:subjects}}"
       multiple="true"
-      dbl10n="true"
```

Comment:
These layouts are useless but `dbl10n="true"` must be kept since subjects is such vocabulary